test/util/setup.js

```diff
@@ -0,0 +1,52 @@
+var loopback = require('loopback');
+
+var Remote = require('../..');
```

Comment:
Ok, I only did it that way because I found it as such before hehe.. didn't know any better. I'll change!